RELEASE.md

```diff
@@ -44,17 +44,18 @@ This document outlines how to create a release of prototool.
     ```
 
 
-5.  Update the version number in internal/vars/vars.go and verify that it
-    matches what is in the changelog.
+5.  Update the version number in internal/vars/vars.go and the Installation
```

Comment:
nit: Looks like you're missing a word here. Consider (or something similar):
```
Update the version number in internal.vars/vars.go, and verify that the changelog matches the version found in the Installation section of the README.md.
```

Reply:
What am i missing? This reads fine to me. This is also the RELEASE.md doc, we don't need prose :-)